src/Constants.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace wisp
+{
+	class Constants
```

Comment:
This doesn't need to be in a class. a namespace should be used since it doesn't create a default constructor.

Reply:
Yup, I accidentally put that commit on the other branch. This will be fixed in the next pull request.